tools/run_tests.ts

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env -S deno run --allow-read --allow-run
+// This script runs the unit tests under thirdparty/deno directory
+
+const p = Deno.run({ cmd: ["node", "tools/skip_tests.cjs"], stdout: "piped" });
+const [_, skipFiles] = await Promise.all([p.status(), p.output()]);
```

Comment:
This could be drastically simplified by making  `tools/skip_tests.cjs` importable from Deno